cwms-data-api/src/main/java/cwms/cda/api/RatingController.java

```diff
@@ -354,8 +357,13 @@ public void getAll(@NotNull Context ctx) {
     public void getOne(@NotNull Context ctx, @NotNull String rating) {
 
         try (final Timer.Context ignored = markAndTime(GET_ONE)) {
-            String officeId = ctx.queryParam(OFFICE);
+
             String timezone = ctx.queryParamAsClass(TIMEZONE, String.class).getOrDefault("UTC");
+            Instant date = null;
+            String specificDate = ctx.queryParam(EFFECTIVE_DATE);
```

Comment:
use `effectiveDate` throughout. it's confusing having this go from EFFECTIVE_DATE, to specificDate, to just "date" in the later code.

Reply:
Updated to use effectiveDate and effectiveDateParam for consistency

Reply:
While consistency is good, I think this illuminates that the parameter is confusing on this endpoint. I recommend using a separate endpoint. I might be missing a use case where a the closest effective date to a parameter is needed, but given temporal interpolation and transitional dates, that seems to be too complicated.